### PR TITLE
Lichmode nerfs

### PR DESCRIPTION
**Pull request by Incoming5643**
tex1_128x128_c6be487244d870d3_14

When you die a string of energy will tie your new body to your old body for a short time, aiding others in determining your location. The duration of this beam scales with the number of deaths you've avoided.

Additionally the post revival stun now also scales in this way.

The spell will also fail if the item and the wizard don't share the same z level, though the nature of space means the odds of the item (or the wizard) looping around back to the station is pretty high.

---

Lich spells by necessity quashed the ability for rounds to end on wizard's death, but now it will properly unquash it once the wizard's body is destroyed.

### CODE REVIEW

**Comment by WJohn** [comment]
You know incoming, if you ever want to make it so the wizard's respawn point isn't just a dark green object and instead want to give it a dedicated sprite, I have a floating black/red urn with TENTACLES here. Maybe make it a cowardly mob that runs away from anything that isn't you?

http://www.mediafire.com/download/c4y6u1x45k49wri/VOLCANO_LAND_1.1.2.rar

**Comment by Incoming5643** [comment]
It's always an option, I haven't forgotten the ideas you pitched.

I just feel like the idea that you can bind it to anything is kind of half the fun of it. It's just a pain that people bind to ridiculously small items like bullet casings or burying it three layers deep in a backpack. This pull will help with that.

 But yeah it couldn't hurt if I could make it look more conspiculous/cooler  than "color it dark green and change the name/desc"

**Comment by Lumipharon** [comment]
What happens if you try ensoul the nuke disk?

**Comment by Incoming5643** [comment]
It's valid, but since the nuke disk respawns if it goes off z-level it's a bad idea to do this as a lich, because the old one is destroyed.

File: code/datums/spells/lichdom.dm [hunk 1]
```diff
@@ -60,8 +64,8 @@
 			charge_max += 600
 			var/mob/old_body = current_body
 			current_body = lich
-			lich.Weaken(10)
-
+			lich.Weaken(10+10*resurrections)
```
Comment (psykzz) [inline code/datums/spells/lichdom.dm]: Is there order here not a problem?

**Comment by WJohn** [comment]
For those curious, this is what the urn I drew looks like.

**Comment by Jalleo** [comment]
Red outline needs to be a bit transparent.

File: code/datums/spells/lichdom.dm (hunk 1 above)
Comment (Incoming5643) [inline code/datums/spells/lichdom.dm]: byond is decent enough about respecting order of operations, yes

**Comment by WJohn** [comment]
Jalleo I literally said it's transparent.

**Comment by KorPhaeron** [comment]
@Incoming5643 

Have you considered making the Resurrection only work if the phylactery is in a station are?

Prevent people from dropping it on a random space tile.

**Comment by Incoming5643** [comment]
there isn't a ton of free space around most stations, and I can't think of any logical (magic logical) reason having the item be in space would automatically disqualify its use. Plus if that happens spacing the phylactery goes from "maybe" to "almost assuredly" ending the wizard.

**Comment by Lumipharon** [comment]
Aren't phylacteries usually vials of blood or something? (I uno), if they are, something like that would probably not fare well in spess. But then, they do have magic so eh.

**Comment by Dorsisdwarf** [comment]
Phylacteries are usually just magical objects, like pendants or chalices.

Also, becoming the wizard's phylactery should increase the w_class so that it'll go into backpacks, but nothing smaller.

**Comment by SweaterKittens** [comment]
@WJohn 

Holy cow that's awesome. I think that would make a great phylactery sprite, but it would be nice if the red outline was transparent. 

@Lumipharon 

I think 'classically' (by that I mean, in DnD) Phylacteries were metal boxes, but they could technically be whatever the lich decided to infuse his soul into.

**Comment by Dorsisdwarf** [comment]
@WJohn I love the sprite but it would be great if the outline was transparent

**Comment by WJohn** [comment]
You guys are assholes sometimes.

**Comment by Nanotransistor** [comment]
Does the lich still die for good if his body is cremated or gibbed?

**Comment by Incoming5643** [comment]
@nanotransistor Yes. That hard to resolve bug turned balance consideration isn't going anywhere so long as minds don't survive the death of their associated brians/bodies

**Comment by Nanotransistor** [comment]
@Incoming5643 Alright, so it's a bug after all, didn't know if it was intended or not.

**Comment by KorPhaeron** [comment]
@Incoming5643 

Could put some weird holder mob inside the phylactery if you want to be hacky maybe?

**Comment by Incoming5643** [comment]
I don't want to be hacky. Also the whole "string of energy connecting new body to old" concept kind of falls apart if you don't have a body to begin with.

**Comment by Razharas** [comment]
@Incoming5643 
Is this ready?

**Comment by Incoming5643** [comment]
yeah man

**Pull request merged**